install_pim/docker/installation_docker.rst

```diff
@@ -206,6 +206,10 @@ If you want to execute only one job:
 
    There is no need to launch a daemon for behat and integration tests. It is performed automatically, the daemon being killed once the test is finished.
 
+.. warning::
+
+   Before stopping or destroying your containers, remember to first stop this daemon if you launched it in background, or you'll end up with a stuck FPM container, and will need to completely restart Docker.
```

Comment:
maybe should you write a word about how to kill it properly with `ps` and `kill`?

Reply:
True enough

Reply:
And done.